packages/providers/src/coin-quantityfy.test.ts

```diff
@@ -0,0 +1,6 @@
+import { coinQuantityfy } from './coin-quantity';
+
+test('coinQuantityfy returns 1 when input is <1', () => {
+  expect(coinQuantityfy([0]).amount.toNumber()).toEqual(1);
+  expect(coinQuantityfy([0.9]).amount.toNumber()).toEqual(1);
+});
```

Comment:
Great addition with this unit test! :+1:

I believe we can make this change to maintain consistency with the majority of our test suites.

```suggestion
describe('coinQuantityfy', () => {
  it('should returns 1 when input is < 1', () => {
    expect(coinQuantityfy([0]).amount.toNumber()).toEqual(1);
    expect(coinQuantityfy([0.9]).amount.toNumber()).toEqual(1);
  });
});
```